### DISCUSSION TITLE
How to attach to anvil or create a private network (for long term testing) ?

### DISCUSSION THREAD

**Discussion by adaki2004** [paradigmxyz/reth#9374] [Help]
Hey there,

I'd like to crete a reth-only based private chain for testing purposes or attach and run it on my local anvil network - which as far as i know is somehow possible.

The currently available documentation [here](https://reth.rs/run/private-testnet.html) about a private network is with Kurtosis, but according to their discord, it is meant to be running for short-term purposes (like a day) but long term test run is not possible/supported.

Is there any method i can either attach this to my anvil OR shoot up the network manually ? Maybe can you provide any CICD scripts, which does that so that i can learn from it ?

Thanks in advance !

**Comment by emhane** [comment]
checkout https://github.com/paradigmxyz/alphanet
